### DISCUSSION TITLE
Webhook doesn't work after replace domain

### DISCUSSION THREAD

**Discussion by ChillMouse** [def-studio/telegraph#299] [Q&A] [answered]
I recently had "example.ru" domain and everything worked

After changing the domain to "bot.another,ru" the webhooks don't work. What do you need? Please give me advice :)

**Comment by fabio-ivona** [comment]
have you tried deleting the webhook and registering a new one?

**Reply by ChillMouse** [reply]
> have you tried deleting the webhook and registering a new one?

Yes

**Reply by fabio-ivona** [reply]
what's the output of the webhook registration command?

**Reply by ChillMouse** [reply]
> what's the output of the webhook registration command?

Sending webhook setup request to: https://api.telegram.org/bot589xxxxxxxxxxxxxx/setWebhook                                                     Webhook updated

**Comment by ChillMouse** [comment, marked as answer]
Config so important....
```php
    'security' => [
        /*
         * if enabled, allows callback queries from unregistered chats
         */
        'allow_callback_queries_from_unknown_chats' => true,

        /*
         * if enabled, allows messages and commands from unregistered chats
         */
        'allow_messages_from_unknown_chats' => true, // <--- this important, before = false

        /*
         * if enabled, store unknown chats as new TelegraphChat models
         */
        'store_unknown_chats_in_db' => true,
    ],
```

And you must have in .env:
```php
APP_URL=https://domain.com

```

**Reply by Natanchik89** [reply]
Thank you very much. There is no such thing in the start manual. When I set the config to true, everything worked.
